src/System.IO.Compression/src/AssemblyInfo.cs

```diff
@@ -0,0 +1,3 @@
+using System.Security;
+
+[assembly: SecurityTransparentAttribute()]
```

Comment:
Given this is coming from the mirror I'm not inclined to change it but we should figure out about trying to make all the files consistent in the repo at some point.